pkg/util/winutil/pdhutil/pdhcounter.go

```diff
@@ -57,43 +58,52 @@ func init() {
 			// buffer's not big enough
 			bufferSize += bufferIncrement
 			continue
+		} else if regerr != nil {
+			return regerr
 		}
 		break
 	}
 	clist := winutil.ConvertWindowsStringList(counterlist)
 	for i := 0; i < len(clist); i += 2 {
 		ndx, _ := strconv.Atoi(clist[i])
-		counterToIndex[clist[i+1]] = ndx
+		counterToIndex[clist[i+1]] = append(counterToIndex[clist[i+1]], ndx)
```

Comment:
There are potentially multiple indices to the same counter name, yes.  And, elsewhere in the code we need all the different indices (because they could map to a different translation.

Reply:
Don't want to hold this PR from merging with the empty struct map (set), but seems like it might make sense to address in the future.